modules/ROOT/pages/veevavault/veevavault-connector-studio.adoc

```diff
@@ -0,0 +1,177 @@
+|*Time Unit* |Select the time unit for the wait time. The default is `MINUTES`.
+|===
++
+*Daily API Limit Exceed Profile:*
++
+The daily API limit exceed profile configuration (*Connector Configuration > Advanced Tab*) to use if the daily API limit of the Veeva Vault API reaches or exceeds the configured threshold. The daily limit is the total number of API calls within any 24-hour window on a daily rolling basis.
++
+If the *Daily API Limit Exceed Profile* is enabled (default), then the connector evaluates the Veeva Vault API's response header (*X-VaultAPI-DailyLimitRemaining*) for the specified value of the threshold, and throws an exception indicating the daily API limit is exceeded. The exception continues to be thrown for each and every job or request created until the daily limit is reset or rolled over to the next day.
++
+Following is an example log recorded which shows warning at *2019-06-06 10:47:31,698*, then an error is thrown at *2019-06-06 10:47:31,699*.
++
+----
+
+WARN 2019-06-06 10:47:31,698 [[MuleRuntime].io.44: [veevavault-connector-demo].get-documents-operationFlow.BLOCKING @108e7f43] [event: 624cfa50-881a-11e9-90e2-d61515f42856] org.mule.extension.veevavault.internal.service.HttpRequestService: Veeva Daily API Limit remaining calls [10000] has exceeded/reached an Daily API Limit Profile threshold set as [10000]. As per API Limit Profile, Veeva Vault Connector will throw an exception.
+
+ERROR 2019-06-06 10:47:31,699 [[MuleRuntime].io.44: [veevavault-connector-demo].get-documents-operationFlow.BLOCKING @108e7f43] [event: 624cfa50-881a-11e9-90e2-d61515f42856] org.mule.extension.veevavault.internal.service.HttpRequestService: Exception caught while processing Sync request ::: https://xxxxx-xxxxx.veevavault.com/api/v19.1/metadata/vobjects/site__v, Exception ::: \{"responseStatus":"FAILURE","errors":[\{"type":"API_LIMIT_EXCEED","message":"The Veeva Vault Connector has reached/exceeded daily API limit remaining calls [10000]. The configured daily api limit threshold is \(10000\)"}\]}
+
+ERROR 2019-06-06 10:47:31,767 [[MuleRuntime].io.44: [veevavault-connector-demo].get-documents-operationFlow.BLOCKING @108e7f43] [event: 624cfa50-881a-11e9-90e2-d61515f42856] org.mule.runtime.core.internal.exception.OnErrorPropagateHandler:
+
+**********************************************************************************
+
+Message : An error occurred from the Veeva Vault API.
+Error Code: API_LIMIT_EXCEED.
+Original Error Message: The Veeva Vault Connector has reached/exceeded daily API limit remaining calls [10000]. The configured daily api limit threshold is (10000).
+Error type : VEEVAVAULT:API_LIMIT_EXCEED
+
+**********************************************************************************
+----
++
+
+[%header%autowidth.spread]
+|===
+|*Field* |*Description*
+|*API Limit Threshold* |Enter the API threshold limit. When the specified limit is reached or exceeded, the connector waits for the specified time. The default is `10000`.
```

Comment:
what is the unit?

Reply:
This is the limit for API requests, so no unit.